### PR TITLE
grammar fixes

### PR DESCRIPTION
**Pull request by tybug**
dont merge yet, I have a couple comments

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ for r in circleguard.verify(1699366, 12092800, 7477458, False):
 
 ### More Generally
 
-The much more flexible way to use circleguard is to make your own Check object and run circleguard with that. This allows for mixing different types of Replay objects - comparing local .osr's to online replays - as well as the liberty to instantiate the Replay objects yourself and use your own Replay subclasses. See [Advanced Usage](#subclassing-replay) for more on subclassing.
+The much more flexible way to use circleguard is to make your Check object and run circleguard with that. This allows for mixing different types of Replay objects - comparing local .osr's to online replays - as well as the liberty to instantiate the Replay objects yourself and use your Replay subclasses. See [Advanced Usage](#subclassing-replay) for more on subclassing.
```
Comment (tybug) [inline README.md]: want to emphasize that the objects are different from ours; we provide Replay subclasses, but they can use their *own* if they so choose.

File: README.md [hunk 2]
```diff
@@ -87,9 +87,9 @@ for r in circleguard.run(Check(replays)):
 
 There are four tiers of options. The lowest option which is set takes priority for any given replay or comparison.
 
-Options can be set at the highest level (global level) by using `Circleguard.set_options`. Options can be changed at the second highest level (instance level) using `circleguard#set_options`, which only affects the instance you call the method on. Be careful to use the static module method to change global settings and the instance method to change instance settings, as they share the same name and can be easy to confuse.
+Options can be set at the highest level (global level) by using `Circleguard.set_options`. Options can be changed at the second-highest level (instance level) using `circleguard#set_options`, which only affects the instance you call the method on. Be careful to use the static module method to change global settings and the instance method to change instance settings, as they share the same name and can be easy to confuse.
```
Comment (tybug) [inline README.md]: hyphenating sounds extremely weird

File: README.md [hunk 3]
```diff
@@ -157,24 +157,24 @@ the corresponding value is RatelimitWeight.LIGHT. If it makes any heavy api call
 corresponding value is RatelimitWeight.HEAVY.
 
 This value is used internally to determine how long the loader class will have to spend loading replays -
-currently LIGHT and NONE are treated the same, and only HEAVY values are counted towards replays to load.
+currently, LIGHT and NONE are treated the same, and only HEAVY values are counted towards replays to load.
```
Comment (tybug) [inline README.md]: nah - adding a pause feels weird here

File: README.md [hunk 4]
```diff
@@ -157,24 +157,24 @@ the corresponding value is RatelimitWeight.LIGHT. If it makes any heavy api call
...
 
-There is no limitation on the order in which replays get loaded; when `circleguard#run(check)` is called, it first checks if `check.loaded` is `True`. If it is, it assumes all the replays in the check object are loaded as well, and moves on to comparing them. Else, it checks if each replay in the check object have `replay.loaded` set to `True` - if so, it moves on to loading the next replay. Otherwise, it calls `replay#load`.
+There is no limitation on the order in which replays get loaded; when `circleguard#run(check)` is called, it first checks if `check.loaded` is `True`. If it is, it assumes all the replays in the check object are loaded as well and moves onto comparing them. Else, it checks if each replay in the check object have `replay.loaded` set to `True` - if so, it moves on to loading the next replay. Otherwise, it calls `replay#load`.
```
Comment (tybug) [inline README.md]: `on to` emphasizes the that it's moving on from a previous action. The reader should take `moves on` as a distinct phrase, where here `onto` is the distinct phrase.

File: README.md [hunk 5]
```diff
@@ -157,24 +157,24 @@ the corresponding value is RatelimitWeight.LIGHT. If it makes any heavy api call
...
 
 ## Contributing
 
-If you would like to contribute to Circleguard, join our discord and ask what you can help with, or take a look at the [open issues for circleguard](https://github.com/circleguard/circleguard/issues) and [circlecore](https://github.com/circleguard/circlecore/issues). We're happy to work with you if you have any questions!
+If you would like to contribute to Circleguard, join our discord and ask what you can help with, or take a look at the open issues for [circleguard](https://github.com/circleguard/circleguard/issues) and [circlecore](https://github.com/circleguard/circlecore/issues). We're happy to work with you if you have any questions!
```
Comment (tybug) [inline README.md]: if we just leave links like this they may think that it's linking to the repo, and not open issues. If we make the first link encompass "open issues" it dispels that notion, even if the second link is just circlecore.

File: README.md (hunk 1 above)
Comment (InvisibleSymbol) [reply README.md]: Just seems like "your own" is a double association - but I will change it back.

File: README.md (hunk 2 above)
Comment (InvisibleSymbol) [reply README.md]: reverted

File: README.md (hunk 3 above)
Comment (InvisibleSymbol) [reply README.md]: reverted

File: README.md (hunk 4 above)
Comment (InvisibleSymbol) [reply README.md]: reverted

File: README.md (hunk 5 above)
Comment (InvisibleSymbol) [reply README.md]: reverted

File: README.md (hunk 1 above)
Comment (tybug) [reply README.md]: you're correct, but it holds emphasis *because* it's a double association

**Comment by tybug** [comment]
cant approve my own pr so lgtm

**Pull request merged**